src/app/api/schedule/route.ts

```diff
@@ -0,0 +1,26 @@
+import type { NextRequest } from "next/server";
+import { NextResponse } from "next/server";
+
+import { db } from "@/lib/db";
+import { gameSessionSchedules } from "@/lib/db/schema";
+
+export async function POST(req: NextRequest) {
+  try {
+    const newGameSession = await req.json();
```

Comment:
Can you use a zod schema to validate this query? Take a look at the examples under `lib/validators`

For insert operations, the `drizzle-zod` package has a method that can generate the schema for you.